docs/docs/application-basics.md

```diff
@@ -50,11 +51,11 @@ But before we dive into this topic, let's have a look at an example:
 }
 ```
 
-What the example above does: before executing the `cmd`, download the resource `https://example.com/app/cool-script.sh` (via Mesos) and make it available in the application task's sandbox. You can check that these have been downloaded by visiting the Mesos UI and clicking into a Mesos worker node's sandbox where you should now find `cool-script.sh`.
+The example above downloads the resource `https://example.com/app/cool-script.sh` (via Mesos) and makes it available in the application task's sandbox before executing the `cmd`. You can check that these have been downloaded by selecting the sandbox of a Mesos worker node in the Mesos UI. You should find `cool-script.sh`.
 
-Note that as of Mesos v0.22 and above the fetcher code now does not make downloaded files executable by default, so `cmd` should be: `chmod u+x cool-script.sh && ./cool-script.sh`.
+Note that as of Mesos v0.22 and above the fetcher code does not make downloaded files executable by default, so `cmd` should be: `chmod u+x cool-script.sh && ./cool-script.sh`. <!-- should we just have this be cmd, then comment on it? wouldn't most people be using the latest Mesos? -->
```

Comment:
Should we just have this be the value of cmd, then comment on it? Would most people be using the latest Mesos?